Project_JS/index.html

```
@@ -0,0 +1,119 @@
+<html lang="en">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)